.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,176 @@
+name: CI
+on: 
+  push:
+    branches:
+      - 'master' 
+      - 'release-*'
+      - '!release-2.8'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        test: ["smoke", "test", "test-e2e", "test-e2e-cron"]
```

Comment:
nice! I think we can lose smoke if we have test-e2e

Reply:
Removed `smoke`